### PR TITLE
Update the memory management values.

### PR DESCRIPTION
**Pull request by bgravenorst**
## Pull Request Description
Added a recommended value when using >8GB of RAM, and added a troubleshooting section.

### CODE REVIEW

File: docs/HowTo/Get-Started/Manage-Memory.md [hunk 1]
```diff
@@ -11,6 +11,9 @@ We recommend setting the maximum heap size to 2GB if you have less than 8GB of R
 more RAM, but want to free up memory for other processes. This results in an approximate process
 size of 3.5GB (on Linux).
 
+If you have more than 8GB of RAM, we recommend setting the maximum heap size to approximately
+5.632GB.
+
```
Comment (rolfyone) [inline docs/HowTo/Get-Started/Manage-Memory.md:16]: if you have 8GB of ram, XXX gb works well. 

If you have more than 8GB of ram, total memory - 4GB if the machine is only running a single instance of teku.
If the machine is used for multiple tasks, set java heap space to 6GB for teku, to limit it's memory usage to approximately 8GB.
that's my attempt, @ajsutton or @mbaxter would potentially have far more accurate answers...

File: docs/HowTo/Get-Started/Manage-Memory.md [hunk 2]
```diff
@@ -11,6 +11,9 @@ We recommend setting the maximum heap size to 2GB if you have less than 8GB of R
 more RAM, but want to free up memory for other processes. This results in an approximate process
 size of 3.5GB (on Linux).
```
Comment (benjaminion) [inline docs/HowTo/Get-Started/Manage-Memory.md:13]: I'd update these for mainnet, and definitely for Pyrmont.

> We recommend setting the maximum heap size to at least 3GB if you have less than 8GB of RAM, or if you have more RAM, but want to free up memory for other processes. This results in an approximate process size of 5GB (on Linux).

File: docs/HowTo/Get-Started/Manage-Memory.md (hunk 1 above)
Comment (benjaminion) [reply docs/HowTo/Get-Started/Manage-Memory.md:16]: I'd say,

> If you have more than 8GB of RAM available, we recommend setting the maximum heap size to 5GB or more.

It's not an exact science, and half gigabytes are neither here nor there.

File: docs/HowTo/Get-Started/Manage-Memory.md (hunk 2 above)
Comment (benjaminion) [reply docs/HowTo/Get-Started/Manage-Memory.md:13]: Also update the `-Xmx2g` to `-Xmx3g` in the examples below.

File: docs/HowTo/Get-Started/Manage-Memory.md (hunk 1 above)
Comment (bgravenorst) [reply docs/HowTo/Get-Started/Manage-Memory.md:16]: Done.

File: docs/HowTo/Get-Started/Manage-Memory.md (hunk 2 above)
Comment (bgravenorst) [reply docs/HowTo/Get-Started/Manage-Memory.md:13]: Done

**Review by rolfyone — APPROVED** [review]
LGTM

**Pull request merged**